experimental/segmenter/src/grapheme.rs

```diff
@@ -27,6 +27,7 @@ pub type GraphemeClusterBreakIteratorUtf16<'l, 's> =
 pub struct GraphemeClusterBreakSegmenter {
     payload: DataPayload<GraphemeClusterBreakDataV1Marker>,
     dictionary: Dictionary,
+    lstm: LstmPayloads,
```

Comment:
Question: why do we need Dictionary and LSTM for GraphemeClusterBreakSegmenter? I thought they were only used for word break and line break.

Reply:
It isn't used for grapheme and sentence.  Rule segmenter handles all UAX29 rules, so I store it in rule segmenter struct. But I can change it to `Option`.